lib/experimental/Information/Avatars/BaseAvatar/index.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  color as AvatarColors,
+  Avatar as AvatarComponent,
+  AvatarFallback,
+  AvatarImage,
+} from "@/ui/avatar"
+import { ComponentProps, forwardRef } from "react"
+
+type ShadAvatarProps = ComponentProps<typeof AvatarComponent>
+
+type Props = {
+  type: ShadAvatarProps["type"]
+  name: string | string[]
+  src?: string
+  size?: ShadAvatarProps["size"]
+  color?: ShadAvatarProps["color"] | "random"
+}
+
+function getInitials(
```

Comment:
this thing worth some unit tests, we have 4 different inputs and 3 different outputs

Reply:
Maybe it should even belong to a separate `utils.ts` file?

Reply:
Moved everything to a `utils.ts` file. What should I test here? No experience with unit tests tbh 😓

Reply:
added tests